wiki/reverse-proxy.md

```diff
@@ -4,11 +4,11 @@
 
 Apache and mod_proxy should not decode/encode slashes and leave them as they are:
 
-````
+````apacheconfig
 <VirtualHost *:80>
-  AllowEncodedSlashes NoDecode
-  ProxyPass /npm http://127.0.0.1:4873 nocanon
-  ProxyPassReverse /npm http://127.0.0.1:4873
+  AllowEncodedSlashes   NoDecode
+  ProxyPass             /npm http://127.0.0.1:4873 nocanon
+  ProxyPassReverse      /npm http://127.0.0.1:4873
```

Comment:
It would not be the same here? with trailing slash at the end?

Reply:
Funny enough, no. If the `url_prefix` is like `http://domain.tld` you do need to add the trailing slash, but when you have a `url_prefix` like `http://domain.tld/npm` it turns to `ProxyPass /npm http://127.0.0.1:4873 nocanon`

I just tested it on the server

Reply:
I think your comment is related with my comment below